docky/cmd/run_open.py

```diff
@@ -70,4 +82,45 @@ class DockyOpen(DockyExec):
 
     def _main(self, *optionnal_command_line):
         super()._main(*optionnal_command_line)
-        self._exec("dcpatched", ["exec", "-e", "NOGOSU=True", self.service] + self.cmd)
+        # self._exec("dcpatched", ["exec", "-e", "NOGOSU=True", self.service] + self.cmd)
+
+        # Get Project Name
+        # Example:     docky-odoo-brasil-14      odoo
+        project_name = self.project.name + "-" + self.project.service
+
+        # Get User
+        user = self._use_specific_user(self.service)
+
+        # Get Container ID
+        command = "docker ps -aqf name=" + project_name
+        # Example of return value
+        # b'b5db9db21381\n'
+        # Option text=true return as string instead of bytes and strip remove break line
+        # TODO: Is there a better way to do it, for example with Plumbum?
+        container_id = subprocess.check_output(command, shell=True,text=True).strip()
+
+        self._exec("docker", ["exec", "-u", user, "-it", container_id, "/bin/bash"])
```

Comment:
When using `docky open --root` user is `False` and this command crashes since it expects strings. 

Something along this line should fix it:

```python
    self._exec("docker", ["exec", "-u", user or 'root', "-it", container_id, "/bin/bash"])
```

Reply:
Fixed, in the method to get the User in order to has a general solution

```bash
    def _use_specific_user(self, service):
        user = self.project.get_user(service)
        if self.root:
            user = "root"
        return user
```

Reply:
You removed the `--service-ports` again though :sweat_smile:

Reply:
Sorry @paradoxxxzero , it was a mistake between the branches, fixed